addons/html_editor/static/src/main/link/link_selection_plugin.js

```diff
@@ -83,7 +59,7 @@ export class LinkSelectionPlugin extends Plugin {
      */
     padLinkWithZwnbsp(link) {
         const cursors = this.dependencies.selection.preserveSelection();
-        this.addFeffs(link, cursors);
+        this.dependencies.feff.addFeffs(link, cursors);
         cursors.restore();
     }
```

Comment:
@dmo-odoo @Zynton @dhba-odoo @deso-odoo Hello!

Just so you know I recently removed the last usage of `padLinkWithZwnbsp` in https://github.com/odoo/enterprise/commit/895c049ea5cb040c5f1e29d7977b8242fabafec0#diff-4fa328c741d7f9dddda830c98f5956d8fbb89674456a7b201bf5eecca285a404

So feel free to remove the deprecated `padLinkWithZwnbsp` in the master forward port :)

Cheers!